Tasks/AzureFileCopy/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 74
+    "Patch": 75
```

Comment:
task.json and task.loc.json do not match. Please checkin the latest version post gulp.